ui/core/components/filters_menu.tsx

```diff
@@ -19,7 +22,36 @@ export class FiltersMenu extends BaseModal {
 	constructor(rootElem: HTMLElement, player: Player<any>, slot: ItemSlot) {
 		super(rootElem, 'filters-menu', { size: 'md', title: 'Filters', disposeOnClose: false });
 
-		let section = this.newSection('Factions');
+		let section = this.newSection('General');
+
+		const ilvlFiltersContainer = <div className="ilvl-filters" />;
+		section.appendChild(ilvlFiltersContainer);
+
+		new NumberPicker(ilvlFiltersContainer as HTMLElement, player.sim, {
```

Comment:
Instead of casting it 2x here and on line 44, I would just cast it here:
```const ilvlFiltersContainer = <div className="ilvl-filters" /> as HTMLElement;```